src/Pickles/Pickles/Runner.cs

```diff
@@ -70,9 +70,8 @@ private static void ApplyTestResultsToFeatures(IContainer container, IConfigurat
         {
             var testResults = container.Resolve<ITestResults>();
 
-            foreach (var node in features)
+            foreach (var featureTreeNode in features.Where(p => p.GetType() == typeof(FeatureNode)).Select(p => (FeatureNode)p))
             {
-                var featureTreeNode = node as FeatureNode;
```

Comment:
this change is actually required because the original code, featureTreeNode = node as FeatureNode will returns null and no test results will be imported